apps/dotcms-ui/src/app/view/components/_common/dot-content-palette/dot-content-palette.component.scss

```diff
@@ -0,0 +1,75 @@
+@use "variables" as *;
+
+:host {
+    width: 15.8rem;
+    display: flex;
+    flex-direction: column;
+    padding: $spacing-1 0;
+}
+
+.dot-content-palette__search {
+    color: $gray;
+    position: sticky;
+    align-self: flex-start;
+    top: 0;
+    width: 100%;
+
+    input {
+        width: 100%;
+        border: none;
+        outline: 0px;
+        background: transparent;
+        padding: $spacing-2 $spacing-5 $spacing-2 $spacing-3;
+        color: $gray;
+    }
+
+    dot-icon {
+        position: absolute;
+        top: 50%;
+        transform: translateY(-50%);
+        right: $spacing-1;
+    }
+}
+
+.dot-content-palette__items {
+
+    overflow: hidden auto;
+
+    a {
+        width: 100px;
```

Comment:
Don't use px make it responsive in a way that if the container width changes it to adjust.